spec/support/features/page_object/topics.rb

```diff
@@ -149,7 +149,7 @@ def someone_updates_topic
 
     def search_for(title)
       fill_in 'Search', with: title
-      find('.search-bar input[type="submit"]').click
+      find('.user-navigation--search input[type="submit"]').click
```

Comment:
Haha this is what happens, I guess, when you work in rails for 5 years as a designer.